package.json

```diff
@@ -78,9 +78,11 @@
     "etcher-latest-version": "^1.0.0",
     "file-type": "^4.1.0",
     "flexboxgrid": "^6.3.0",
+    "gpt": "^1.0.0",
     "immutable": "^3.8.1",
     "lodash": "^4.5.1",
     "lzma-native": "^1.5.2",
+    "mbr": "^1.1.1",
```

Comment:
I guess that'd be https://github.com/jhermsmeier/node-disk, but since we're doing very specific stream slicing here, I don't think that'd be a good idea (just another minor abstraction to maintain), especially when considering that most of this will be done quite differently soon-ish.